File: Backend/EventManager.Domain/Models/EmployeeGroup.cs
using EventManager.Domain.Enums;
using System;

namespace EventManager.Domain.Models
{
    public class EmployeeGroup
    {
        public int Id { get; set; }
        public int Amount { get; set; }
        public PositionType PositionType { get; set; }
        public DateTime GatheringTime { get; set; }
        public Address GatheringAddress { get; set; }
        public double salary { get; set; }
        public double Hours { get; set; }
    }
}

File: Frontend/event-manager/src/app/shared/models/employee-group.ts
import { Address } from './address';
import { Data } from '@angular/router';
import { PositionType } from './../enums/position-type';
export class EmployeeGroup {
  id: number;
  amount: number;
  positionType: PositionType;
  gatheringTime: Data;
  gatheringAddress: Address;
  salary: number;
  hours: number;

  constructor(id: number, amount: number, positionType: PositionType,
    gatheringTime: Data, gatheringAddress: Address, salary: number, hours: number
  ){
    this.id = id;
    this.amount = amount;
    this.positionType = positionType;
    this.gatheringTime = gatheringTime;
    this.gatheringAddress = gatheringAddress;
    this.salary = salary;
    this.hours = hours;
  }
}
